helix-view/src/editor.rs

```diff
@@ -88,6 +88,12 @@ impl Editor {
     pub fn switch(&mut self, id: DocumentId, action: Action) {
         use crate::tree::Layout;
         use helix_core::Selection;
+
+        if !self.documents.contains_key(id) {
+            log::warn!("cannot switch to document that does not exist (anymore)");
```

Comment:
Let's make this `log::error!`, ideally this shouldn't happen. We'll need to add something like `forget_doc` on the view to remove it from both alternate file and the jumplist: https://github.com/mawww/kakoune/blob/c507863a00289fe4b210ecc4fef5554de4c87bf4/src/context.cc#L197-L210